Source/LinuxParking.API/appsettings.Development.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+
+  "ConnectionStrings": {
+    "Default": "host=localhost;port=5432;database=linuxpark;user id=root;password=secret"
```

Comment:
Having passwords exposed in public repo might be a problem in other kinds of projects :)

Reply:
Good point! Will make sure to use a more secure approach next time.